README.md

```diff
@@ -1,3 +1,42 @@
+# Datadog Fork
+
+This is an experimental fork of [elastic/otel-profiling-agent](https://github.com/elastic/otel-profiling-agent). The upstream project is in the process of being [donated](https://github.com/open-telemetry/community/issues/1918) to the OpenTelemetry project. Please refer to our [documentation](https://docs.datadoghq.com/profiler/) for a list of offically supported Datadog profilers.
+
+Our fork adds support for sending profiling data to the Datadog backend via the Datadog Agent. We are active members of the OpenTelemetry Profiling SIG that is working on the OpenTelemetry profiling signal. However, the signal is still under active development, so this fork can be used by Datadog users until we release our support for directly ingesting the data using OTLP.
+
+## Build
+
+To build the agent, you can use the following commands:
+
+```
+make docker-image
+make agent
+```
+
+This will create a `otel-profiling-agent` binary in the current directory.
+
+Alternatively you can download pre-built amd64 and arm64 binaries from a recent [CI release run](https://github.com/DataDog/otel-profiling-agent/actions/workflows/release.yml) by selecting a recent run and then scrolling down to the "Artifacts" section.
```

Comment:
I see there are some tags now as well. Should I point people to those instead? Will the `dev` tag always be the recommended one? Do we overwrite it?

Reply:
`dev` is moved to always point to the latest main build. I don't think we should recommend this one.
I will do a proper release once this PR is merged.
We should point to the latest non pre-release in `https://github.com/DataDog/otel-profiling-agent/releases`.
About the versioning, what something like `0.0.1-dd` to clearly identify it as a Datadog version ?